scripts/nextcloud.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python2
+""" Takes nextcloud settings as yaml an sets them accordingly with php occ """
+from subprocess import call
+import argparse
+import yaml
+
+
+def get_type_php_from_object(value):
+    """ Return php type name of type of value """
+    if isinstance(value, bool):
+        return "boolean"
+    elif isinstance(value, int):
+        return "integer"
+    elif isinstance(value, float):
+        return "float"
+    return "string"
+
+
+def generate_paths(data):
+    """ Return list of lists containing the required values to pass to call """
+    execute_list = []
+    if not isinstance(data, list) and not isinstance(data, dict):
```

Comment:
why the second condition? Are dicts also being detected as lists in python?

Reply:
because when you get something else than a dict or a list you want to return just the data

Reply:
Okay, I overread the first `not`.